packages/web-react/src/components/Tabs/TabLink.tsx

```diff
@@ -6,12 +6,12 @@ interface TabLinkProps extends ChildrenProps, TransferProps {
   href: string;
 }
 
-const TabLink = ({ children, href }: TabLinkProps): JSX.Element => {
+const TabLink = ({ children, href, ...restProps }: TabLinkProps): JSX.Element => {
   const { classProps } = useTabsStyleProps();
 
   return (
     <li className={classProps.item}>
-      <a href={href} className={classProps.link} role="tab">
+      <a href={href} className={classProps.link} role="tab" {...restProps}>
```

Comment:
We do not want to replace accidentally attributes that are set by us.
```suggestion
      <a  {...restProps} href={href} className={classProps.link} role="tab">
```

Reply:
fixed